ticketbuyer.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/decred/dcrd/chaincfg/chainhash"
+	"github.com/decred/dcrutil"
+	"github.com/decred/dcrwallet/ticketbuyer"
+	"github.com/decred/dcrwallet/waddrmgr"
+	"github.com/decred/dcrwallet/wallet"
+	"github.com/decred/dcrwallet/wtxmgr"
+)
+
+// startTicketPurchase launches ticketbuyer to start purchasing tickets.
+func startTicketPurchase(w *wallet.Wallet, ticketbuyerCfg *ticketbuyer.Config) {
+	retryDuration := time.Second * 5
+	for {
+		if w.ChainClient() != nil {
+			break
+		}
+		time.Sleep(retryDuration)
+		tkbyLog.Debugf("Retrying chain client connection in %v", retryDuration)
+	}
+	dcrdClient := w.ChainClient().Client
+	walletCfg := &ticketbuyer.WalletCfg{
```

Comment:
oh, I see.  So dcrticketbuyer is still doing all the purchasing logic then.  Makes sense because there's no rpc interface yet to start the purchaser or change its parameters.  Once that is in I think this can be removed.